### ISSUE TITLE
Simple pixel ruler

### ISSUE THREAD

**Issue by saturasvati** [flameshot-org/flameshot#3178]
### Feature Description

Ruler was [already offered](https://github.com/flameshot-org/flameshot/issues/2948), but i think that too difficult and hard for so simple tool. 
Please, add ruler that should measurement just width, height and length in px, how it is in firefox develop tools. And it be very nice, if that will have even protractor. 

![изображение](https://user-images.githubusercontent.com/56877035/233858848-de1b092a-7a19-46f6-b84e-2bef3ba3d5fe.png)

**Comment by mmahmoudian** [comment]
well, technically this is already implemented:

[Peek 2023-04-24 10-35.webm](https://user-images.githubusercontent.com/390889/233929568-c60caab2-ee3b-4e96-ada7-d3d74da7880e.webm)

Doesn't this already do what you are asking for?

**Comment by saturasvati** [comment]
Yes, it is. But unfortunately I use it on Windows. I don't find this in the GUI settings, and the CLI argument doesn't work.

In any case, FS displays the width and height of the selected rectangle, and the problem is partially irrelevant.

**Comment by mmahmoudian** [comment]
> But unfortunately I use it on Windows. I don't find this in the GUI settings, and the CLI argument doesn't work.

We have #2118 for implementing CLI for Windows as well. 

> FS displays the width and height of the selected rectangle

Yes, Flameshot shows that already. Combined with the magnifications we have, it is actually very convenient to measure things perfectly. The only time I have problem is when the selection size is too small.

I don't know if we should keep this issue open. I think it is good if you can comment in #2948 so that it gets read when someone is trying to implement that feature. Would be extra nice if you can also suggest how this should be implemented along side and how user can use it.
